jwst/datamodels/tests/test_models.py

```diff
@@ -49,6 +49,95 @@ def teardown():
     shutil.rmtree(TMP_DIR)
 
 
+@pytest.fixture(scope='module')
+def empty_model():
+    """Create an empy model"""
+    return DataModel()
```

Comment:
This is potentially dangerous to scope this to "module", unless there's a good reason. Better to leave the scope at function, so there's not any unintended effects on one test from another, and dependency on run order.